tasks/install.xml

```diff
@@ -180,20 +181,31 @@
             </filterchain>
         </copy>
 
-        <!-- If we're using the-vagrant, offer to install Drupal. -->
-        <if>
-            <equals arg1="${env.PALANTIR_ENVIRONMENT}" arg2="vagrant" />
-            <then>
-                <input propertyName="install_now" validArgs="y,n" message="Install Drupal now" promptChar="?" />
+        <!-- Run the first build. -->
+        <phing dir="${application.startdir}" target="build" inheritAll="false" />
+
+        <!-- If Drupal has a database connection available, offer to install Drupal. -->
+        <trycatch>
+            <try>
+                <!-- The 'haltonfailure' attribute seems to have the opposite behavior
+                     from what is described in the Phing documentation. -->
+                <phing dir="${application.startdir}" target="drupal-has-database-connection" inheritAll="false" haltonfailure="true" />
```

Comment:
When running `the-build-installer` outside the VM, it now fails with "Drupal database connection not available" because of the try/catch inside the `drupal-has-database-connection` task.

Reply:
Oops, it shouldn't fail, it should just not do that part of the install. Let me check that out.

Reply:
I've added a message/output to clarify what's happening here -- it's just skipping the Drupal install, but the rest of the-build is already installed.